ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/checkoutform/.content.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:description="This is Check Out Form"
```

Comment:
@nitheshkumar-hcl  update more details in description.

Reply:
updated the details in  description